lib/cf_app_discovery/service_broker.rb

```diff
@@ -54,9 +54,13 @@ class ServiceBroker < Sinatra::Application
     delete "/v2/service_instances/:instance_id/service_bindings/:id" do |_, binding_id|
       content_type :json
       service_binding = client.service_binding(binding_id)
-      app_guid = service_binding.fetch(:entity).fetch(:app_guid)
-      cleaner = Cleaner.new(filestore_manager: filestore_manager)
-      cleaner.remove_old_target(app_guid)
+      if service_binding.has_key?(:entity) == false
+        STDERR.puts "service binding: #{service_binding}"
```

Comment:
this looks like debug output - can we remove (or make it into a proper error message)

Reply:
Will try to output a proper error message as I was occasionally getting errors on deletion.